Description: Don't build ib_qib
 Adds kernel headers that need to be modified for the OPA build. The source was
 linux-3.16.39. Modification will be performed by sucessive patches.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Forwarded: not-needed
Last-Update: <2017-12-14>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,8 @@
 
 CFLAGS_MODULE += -DUSE_PI_LED_ENABLE=1
 
-obj-y := rdmavt/ hfi1/ ib_qib/
+obj-y := rdmavt/ \
+	hfi1/
 
 else
 #normal makefile
@@ -22,7 +23,7 @@
 	$(MAKE) -C $(KDIR) M=$$PWD clean
 
 install:
-	$(MAKE) INSTALL_MOD_DIR=updates -C $(KDIR) M=$$PWD modules_install
+	$(MAKE) INSTALL_MOD_PATH=$(DESTDIR) INSTALL_MOD_DIR=updates/ifs-kernel-updates -C $(KDIR) M=$$PWD modules_install
 
 endif
 
